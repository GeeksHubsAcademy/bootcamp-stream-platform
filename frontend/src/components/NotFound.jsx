import React from 'react';

import { Link } from '@reach/router';

const NotFound = () => (
  <section className='404'>
    <h1>Pel√≠cula no encontrada</h1>
    <h2>404</h2>
    <Link to="/movies/popular">Go to popular</Link>
  </section>
);

export default NotFound;
