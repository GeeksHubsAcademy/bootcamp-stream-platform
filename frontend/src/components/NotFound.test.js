import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';

import NotFound from './NotFound';

test('Deberia renderizar Error 404 segun su snapshot', () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<NotFound />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
