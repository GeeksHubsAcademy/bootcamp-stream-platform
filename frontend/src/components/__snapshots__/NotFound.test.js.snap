// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deberia renderizar Error 404 correctamente 1`] = `
<section
  className="404"
>
  <h1>
    Página no encontrada
  </h1>
  <h2>
    404
  </h2>
  <ForwardRef
    to="login"
  >
    Logeate
  </ForwardRef>
</section>
`;

exports[`Deberia renderizar Error 404 segun su snapshot 1`] = `
<section
  className="404"
>
  <h1>
    Página no encontrada
  </h1>
  <h2>
    404
  </h2>
  <ForwardRef
    to="login"
  >
    Logeate
  </ForwardRef>
</section>
`;
